client/lib/abtest/active-tests.js

```diff
@@ -83,4 +92,14 @@ export default {
 		defaultVariation: 'control',
 		allowExistingUsers: true,
 	},
+	newUsersWithFreePlan: {
+		datestamp: '20201218',
+		variations: {
+			newOnboarding: 50,
+			control: 50,
+		},
+		localeTargets: nonENLanguageSlugs,
```

Comment:
To simplify, can we use `localeExceptions`? We can then also remove the definition of `nonENLanguageSlugs`.

```suggestion
		localeExceptions: [ 'en' ]
```

Reply:
Thanks for the hint! I've missed that option somehow 😂

Reply:
When `localeTargets` is not specified in `active-tests`, it will only target `en` users. See: https://github.com/Automattic/wp-calypso/blob/6ec65ffad8107678588f9fc5b917de530cf646b7/client/lib/abtest/index.js#L129-L130

To avoid using `localeTargets: nonENLanguageSlugs` where the whole list of language is imported, one can use `localeTargets: 'any'` in combination with `localeExceptions: [ 'en', 'en-gb', 'es' ]`. See: https://github.com/Automattic/wp-calypso/blob/6ec65ffad8107678588f9fc5b917de530cf646b7/client/lib/abtest/index.js#L131-L134